changelog.mdx

```diff
@@ -7,6 +7,55 @@ description: Updates to Chalk!
 
 ---
 
+## October 7, 2024
+
+### Underscore expressions support datetime subtraction and total_seconds
+
+[Underscore expressions](/docs/underscore) now support datetime subtraction and the use of a new
+library function `chalk.functions.total_seconds`. This allows you to compute the number of seconds
+in a time duration and define more complex time interval calculations using performant underscore
+expressions.
+
+For example, to define a feature that computes the difference between two date features in days and weeks,
+we can use `chalk.functions.total_seconds` and underscore date expressions together.
+```python
+from chalk.functions as F
+from chalk.features import _, features, Primary
+@features
```

Comment:
Can you make this a meaningful example? This is a good opportunity for us to showcase use cases. Example might be "days since account opening"

Reply:
yes